src/main/java/com/ericsson/ei/controller/AggregatedObjectController.java

```diff
@@ -6,29 +6,39 @@
 import com.ericsson.ei.controller.model.QueryBody;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
 
 /**
- * This class retrieves aggregated objects given a free style query. Criteria is required to have in the request body, while options and filter are optional.
+ * This class returns aggregated data given a specific ID of an aggregated object.
  * (Generated with springmvc-raml-parser v.2.0.5)
  * 
  */
 @RestController
 @Validated
-@RequestMapping(value = "/query", produces = "application/json")
-public interface QueryController {
+@RequestMapping(value = "/aggregated-objects", produces = "application/json")
+public interface AggregatedObjectController {
 
 
+    /**
+     * This method retrieves aggregated data on a specific aggregated object, given an ID.
+     * 
+     */
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET)
```

Comment:
In other places like subscription this changed to parameters, maybe this should also be a parameter so that adress would be host:port/aggregated-objects?id=xxx

Reply:
It was URI parameter before, but in the story it was asked to be changed in the current form.

Reply:
Yes that is true. For single elements in a collection it has been changed to /<collection-name>/<id> and for multiple it has been changed to filter style query like this /<collection-name>?<key>=<values> but in this case we only have single queries. So /<collection-name>/<id> should be the way.